@here/harp-mapview/lib/VisibleTileSet.ts

```diff
@@ -132,13 +135,76 @@ export interface DataSourceTileList {
     /**
      * List of tiles we want to render but they might not be renderable yet (e.g. loading).
      */
-    visibleTiles: Tile[];
+    visibleTiles: Array<Tile | TileProxy>;
 
     /**
      * List of tiles that will be rendered. This includes tiles that are not in the visibleTiles
      * list but that are used as fallbacks b/c they are still in the cache.
      */
-    renderedTiles: Tile[];
+    renderedTiles: Array<Tile | TileProxy>;
+}
+
+/**
+ * Creates a unique key based on the supplied parameters. Note, the uniqueness
```

Comment:
Please fill the line length of 100 chars in all comments.

Reply:
Done